### ISSUE TITLE
GPU gets counted for each worker 

### ISSUE THREAD

**Issue by nyck33** [dask/distributed#7884]
As suggested by your moderator on the forums:

https://dask.discourse.group/t/why-am-i-seeing-16-gib-memory-when-i-have-an-nvidia-geforce-gtx-1650-gpu/1891

**Comment by jacobtomlinson** [comment]
Yeah this double counting is kinda expected when not using `dask-cuda`. `LocalCluster` will start one worker per CPU core, and each worker will be able to see the GPU. We could make things more intelligent in distributed but then we are getting close to duplicating what `dask-cuda` does.

We could potentially collect the UUID of the GPU and pass that back to the scheduler, then do some deduplication. But I'm not sure how beneficial that would be because users using `LocalCluster` probably aren't using the GPUs, and if they did want to use the GPUs they would use `LocalCUDACluster` instead which works as expected.
